src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,68 @@
+package mate.academy.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
+    private List<Ticket> tickets;
+    private LocalDateTime orderDate;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private User user;
```

Comment:
The relationship between Order and User entities is not clearly defined. Consider adding 'cascade' attribute in @ManyToOne annotation and specify CascadeType as needed.